### ISSUE TITLE
ERROR with patchapk in the newest objection version.

### ISSUE THREAD

**Issue by TheFly1ngDutch** [sensepost/objection#134]
Hi, 

I'm trying to use objection patchapk and i end up getting the following error.
Even using --skip-resources has not solved the issue. Can you advice please. 
Thanks in advance.

![image](https://user-images.githubusercontent.com/22275030/44932306-7e3bc480-ad82-11e8-9d77-caa31891ea4b.png)

![image](https://user-images.githubusercontent.com/22275030/44932384-db377a80-ad82-11e8-9cc4-d9da855966bf.png)

**Comment by leonjza** [comment]
Unfortunately, in this case the patching failed because actually, apktool failed to rebuild the app. You can try and manually follow the unpacking, repacking process and if that works, manually patching the target application and repacking. If its something small you need to tweak to get apktool to rebuild the app post patching with objection, you can add the `--pause` flag to give you time to manually fix things up.

tl;dr, apktool errors are not really something objection can automatically recover from, and instead require manual patching.

Good luck!
